src/components/templates/TalkShowStudio/TalkShowStudio.tsx

```diff
@@ -47,9 +32,19 @@ export const TalkShowStudio: FC<TalkShowStudioProps> = ({ venue }) => {
   const stage = useStage();
   const { userId, profile } = useUser();
   const currentVenue = useSelector(currentVenueSelectorData);
-  const remoteUsers = useAgoraRemotes({ client: remotesClient });
   const isRequestToJoinStageEnabled = venue.requestToJoinStage;
 
+  const [remotesClient, screenClient, cameraClient] = React.useMemo(() => {
+    return Array.from({ length: 3 }).map(() =>
+      AgoraRTC.createClient({
+        codec: "h264",
+        mode: "rtc",
+      })
+    );
```

Comment:
Do these clients need to be 'closed' or otherwise marked as being no longer needed in some form? If so, should that be happening in the 'cleanup' return of this `useEffect` (it probably should be)

Reply:
@0xdevalias fixed in efa8913141c5d517382f8f25178d4967c902a3df